### DISCUSSION TITLE
how to write list validate

### DISCUSSION THREAD

**Discussion by langziyang** [mantinedev/mantine#5170] [Q&A]
on doc, i see the list is object,[{},{}],but if my list is just string [string,string],the how to write validate

**Comment by rtivital** [comment]
```tsx
useForm({
  validate: {
    arrayOfStrings: (value) => {
      // perform your validation here
    }
  }
})
```

**Reply by langziyang** [reply]
![image](https://github.com/mantinedev/mantine/assets/2345387/b48540fa-bc6b-4666-83b9-a577dd5b8e54)
sorry,how validate the second is empty and show error

**Reply by rtivital** [reply]
You need to use objects in this case, raw string arrays are not considered lists
